doc/source/reference/development/design.rst

```diff
@@ -18,7 +18,7 @@ a Python interpreter.
 
 In order to achieve this goal, **freud** takes the following viewpoints:
 
-* **freud** works directly with `NumPy <http://www.numpy.org/>_`
+* **freud** works directly with `NumPy <https://numpy.org/>`__
```

Comment:
@klywang This is a tiny unrelated fix. I noticed this link wasn't rendering correctly while you were showing the docs during our call today.